Readme.md

```diff
@@ -1,54 +1,54 @@
-
```

Comment:
There is something real bad happening with my git. Execuse me if I give review on stale commits. Or repeat my comments.

Reply:
All good, I am learning a lot doing this so I don't know what you would need to excuse yourself for anyways.